opencraft/settings.py

```diff
@@ -261,6 +261,7 @@
 OPENSTACK_SANDBOX_SSH_KEYNAME = env('OPENSTACK_SANDBOX_SSH_KEYNAME', default='opencraft')
 OPENSTACK_SANDBOX_SSH_USERNAME = env('OPENSTACK_SANDBOX_SSH_USERNAME', default='ubuntu')
 
+OPENSTACK_PRODUCTION_FLAVOR = env.json('OPENSTACK_PRODUCTION_FLAVOR', default={"ram": 8192, "disk": 80})
```

Comment:
@lgp171188 Should this default to `{"name": "vps-ssd-3"}`?

Reply:
@itsjeyd, the default value for the existing `OPENSTACK_SANDBOX_FLAVOR` is set to `{"ram": 4096, "disk": 40}` and that is why I setup the default value similarly for the production instance flavor. Should I still change it to `vps-ssd-3`? `vps-ssd-3` seems to be specific to a particular OpenStack provider and I think it is best defined in the `.env` file.

Reply:
@lgp171188 

> ... the default value for the existing `OPENSTACK_SANDBOX_FLAVOR` is set to `{"ram": 4096, "disk": 40}` and that is why I setup the default value similarly for the production instance flavor.

Ah, I see -- makes sense to go with the original approach, then.

A follow-up question, though: Does `vps-ssd-3` come with a disk that's twice the size of `vps-ssd-2`? I'm asking because we really only wanted to increase the amount of RAM that each production instance has available, not necessarily partition size.

Reply:
@itsjeyd, `vps-ssd-2` has 4 GB RAM with 1 vCPU and 20 GB of disk space and `vps-ssd-3` has 8 GB RAM with 2 vCPUs and 40 GB of disk space. I don't see a flavor which just offers more RAM.